how-it-works/2_chainkeytech/01_chain-key-technology.card.md

```diff
@@ -1,12 +1,24 @@
 ---
-title: Chain key cryptography
+title: Chain-key cryptography
 ---
 
 ![](/img/how-it-works/chain-key-technology.600x300.jpg)
 
-# Chain key cryptography
+# Chain-key cryptography
```

Comment:
```suggestion
# Chain-Key Cryptography
```

Reply:
But wouldn't that make it inconsistent with the capitalization rules we use elsewhere?

Reply:
<img width="669" alt="image" src="https://user-images.githubusercontent.com/14757975/199953838-55796871-098c-43ec-b7b8-f303390e2823.png">
your call. It's capitalised further down the page again as well